web/libs/ui/src/lib/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable-next-line */
```

Comment:
why do we ignore it? we can just make two imports if that's the case

Reply:
Hmm, not sure why this is ignored, really shouldn't be.